pdns/dnsdistdist/dnsdist.service.in

```diff
@@ -48,6 +48,11 @@ RestrictRealtime=true
 RestrictSUIDSGID=true
 SystemCallArchitectures=native
 SystemCallFilter=~ @clock @debug @module @mount @raw-io @reboot @swap @cpu-emulation @obsolete
+ProtectProc=invisible
+MemoryDenyWriteExecute=true
+PrivateIPC=true
+RemoveIPC=true
+DevicePolicy=closed
```

Comment:
It looks like `DevicePolicy=closed` is already enabled by the stricter `PrivateDevices=true` above.